public/components/homepage.js

```diff
@@ -86,23 +88,50 @@ class FreelancerCreateBtn extends React.Component {
 
 // Card which displays a freelancer's information
 class FreelancerCard extends React.Component {
+  formatAvgScore(avgScore) {
+    if (isNaN(avgScore)) {
+      return '-';
+    } else {
+      return avgScore;
+    }
+  }
+  formatPrice(price) {
+    if (price) {
+      return price.min + ' - ' + price.max + ' CHF';
+    } else {
+      return '-';
+    }
+  }
+  formatDistance(distance) {
+    if (!distance || distance !== Number.MAX_SAFE_INTEGER) {
```

Comment:
I'm not sure about this condition. If there is no distance, why format it at all? And if there is, a distance, then return `-`?